src/zcash/address/zip32.h

```diff
@@ -136,8 +348,109 @@ struct SaplingExtendedSpendingKey {
     }
 };
 
+class UnifiedSpendingKey;
+class UnifiedFullViewingKey;
+
+class ZcashdUnifiedAddress {
```

Comment:
So, the one place that I thought it would be a good idea to use the unified key address space is in the Sapling migration tool - basically, to have migration to Sapling use the UA-derived Sapling address rather than the one associated with the legacy account.